jwt.js

```diff
@@ -127,6 +169,10 @@ module.exports = ({
                 tokenCache,
                 request => request.headers.authorization && request.headers.authorization.match(/^basic\s+(.+)$/i)?.[1]
             ));
+            server.auth.scheme('custom', customAuthChecker(
```

Comment:
isn't it better to call it `signatureAuthChecker`? Maybe more schemes will appear in the future. `custom` sounds too broad while it looks for `Authorization: Signature ...`